### PR TITLE
Task/#169_document-javascript-sdk

### LINKED ISSUE #169: Document JavaScript SDK
Add/Remove documentation related to the new features of the platform.

### PR DESCRIPTION
**Pull request by alvaro9210**
- Closes #169 - Document JavaScript SDK
- Added comments to Object Model page
- Added documentation for Access Points
- Added example to Object Schema page
- Added documentation for Real-time capabilities

### CODE REVIEW

File: source/documentation/v1.0.x/javascript-sdk/object-model/index.ejs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: documentation/document
 title: Object Model
-description: An in depth description of the Carbon Object Model.
+description: What is an Access Point and how to use them.
```
Comment (MiguelAraCo) [inline source/documentation/v1.0.x/javascript-sdk/object-model/index.ejs]: ?

File: source/documentation/v1.0.x/javascript-sdk/object-model/index.ejs [hunk 2]
```diff
@@ -65,10 +71,12 @@ version: v1.0.x
 			or <code>https</code> scheme (depending on the platform's configuration).
 		</p>
 		<p>
-			Using URIs as identifiers allow Carbon documents to be <strong>unique</strong>, <strong>resolvable</strong> and <strong>relative to
-				the network</strong>.
+			Using URIs as identifiers allow Carbon documents to be <strong>unique</strong>, <strong>resolvable</strong> and
+			<strong>relative tothe network</strong>.
```
Comment (MiguelAraCo) [inline source/documentation/v1.0.x/javascript-sdk/object-model/index.ejs]: typo

File: source/documentation/v1.0.x/javascript-sdk/object-schema/index.ejs [hunk 3]
```diff
@@ -161,35 +187,219 @@ version: v1.0.x
...
+				carbonA.extendObjectSchema( &#123;
 					"content": &#123;
 						"@type": "string"
 					&#125;
 				&#125; );
 			</code></pre>
-			
-		<p>If the application A has its object schema set this way it won't have a problem. The SDK will filter any "content" non string value when retrieving the project.</p>
+
+		<p>If application <strong>A</strong> has its object schema defined this way, there won't have a problem. The SDK will filter any "content" non string value when retrieving the project.</p>
```
Comment (MiguelAraCo) [inline source/documentation/v1.0.x/javascript-sdk/object-schema/index.ejs]: it won't have a problem

File: source/_data/documentation.yml [hunk 4]
```diff
@@ -146,7 +146,10 @@ versions:
             description: Not yet documented
           - title: Access points
             link: access-points
-            description: Not yet documented
+            description: What is an Access Point and how to use them.
+          - title: Real-Time capabilities
+            link: real-time-capabilities
+            description: Discover the capabilities of a real time linked data platform.
```
Comment (mhernandeza) [inline source/_data/documentation.yml]: real-time

File: source/documentation/v1.0.x/javascript-sdk/access-points/index.ejs [hunk 5]
```diff
@@ -16,16 +15,671 @@ version: v1.0.x
...
-<p>
-	We are doing our best to catch up quickly with the documentation. If you have questions regarding
-	this topic you can always ask us directly through our community. See how <a href="/community-and-support">here</a>
-</p>
+
+<!-- START: Introduction -->
+<section class="mainContent-section">
+	<h2 class="hidden">Introduction</h2>
+	<p>
+		One of the main goals of a Linked Data Platform is to link data (obviously). Such links could be made within a
```
Comment (mhernandeza) [inline source/documentation/v1.0.x/javascript-sdk/access-points/index.ejs]: Such links can exist within a single domain or between different domains.

File: source/documentation/v1.0.x/javascript-sdk/real-time-capabilities/index.ejs [hunk 6]
```diff
@@ -0,0 +1,1000 @@
...
+
+</section>
+<!-- END: Using the Documents Service -->
+
+
+<!-- START: Messaging Service -->
+<section class="mainContent-subSection">
+	<h3 class="hidden">Messaging Service</h3>
+	<p>
+		We just saw how to subscribe to an event and execute a function depending on the event we subscribed to,
```
Comment (mhernandeza) [inline source/documentation/v1.0.x/javascript-sdk/real-time-capabilities/index.ejs]: We just saw how to subscribe to an event and execute a function depending on the event we subscribed to. If you noticed, ...

File: source/documentation/v1.0.x/javascript-sdk/real-time-capabilities/index.ejs [hunk 7]
```diff
@@ -0,0 +1,1000 @@
...
+			</div>
+		</div>
+
+	</div>
+
+
+	<p>
+		Whenever you subscribe to an event, if you haven't called the <code>connect</code> or <code>reconnect</code> methods, the SDK will automatically
+		connect to the <span class="highlight">broker/</span> endpoint and will actively poll that connection. That gives you the flexibility to just dive into
+		code the subscriptions without worrying about opening and closing connections.<br>
```
Comment (mhernandeza) [inline source/documentation/v1.0.x/javascript-sdk/real-time-capabilities/index.ejs]: ~code~ coding

File: source/documentation/v1.0.x/javascript-sdk/real-time-capabilities/index.ejs [hunk 8]
```diff
@@ -0,0 +1,1000 @@
...
+	</p>
+</section>
+<!-- END: Messaging Service -->
+
+
+<!-- START: Conclusion -->
+<section class="mainContent-section">
+	<h2>Conclusion</h2>
+	<p>
+		The SDK immediately notifies whenever an event occurs thanks to subscriptions.<br>
```
Comment (mhernandeza) [inline source/documentation/v1.0.x/javascript-sdk/real-time-capabilities/index.ejs]: Who does the SDK notify? You, your application ..?

File: source/documentation/v1.0.x/javascript-sdk/real-time-capabilities/index.ejs [hunk 9]
```diff
@@ -0,0 +1,1000 @@
...
+		A subscription is formed of a <u>subject</u> (<span class="highlight">document</span>), and <u>event</u> and and <u>action</u> to trigger when the event occurs.<br>
```
Comment (mhernandeza) [inline source/documentation/v1.0.x/javascript-sdk/real-time-capabilities/index.ejs]: A subscription is formed of a <u>subject</u> (<span class="highlight">document</span>), an <u>event</u>, and an <u>action</u> to trigger when the event occurs.<br>

File: source/documentation/v1.0.x/javascript-sdk/real-time-capabilities/index.ejs [hunk 10]
```diff
@@ -0,0 +1,1000 @@
...
+	</p>
+	<p>
+		A subscription can be created using the <code>on</code> method of the <span class="highlight">PersistedDocument.Class</span> or the <span class="highlight">documents</span> service.<br>
+		The difference between <span class="highlight">PersistedDocument.Class</span> and the <span class="highlight">documents</span> service is that <span class="highlight">documents</span> service
+		lets you specify <a href="https://ant.apache.org/manual/dirtasks.html" target="_blank">Ant patterns</a> to subscribe to multiple levels of an endpoint whereas the
+		<span class="highlight">PersistedDocument.Class</span> don't.<br>
```
Comment (mhernandeza) [inline source/documentation/v1.0.x/javascript-sdk/real-time-capabilities/index.ejs]: ~don't~ doesn't

File: source/documentation/v1.0.x/javascript-sdk/real-time-capabilities/index.ejs [hunk 11]
```diff
@@ -0,0 +1,1000 @@
...
+	</p>
+	<p>
+		The <code>Carbon.Messaging.Service</code> is the responsible to handle the connections to the <span class="highlight">broker/</span> endpoint of the platform and with it, you can configure
```
Comment (mhernandeza) [inline source/documentation/v1.0.x/javascript-sdk/real-time-capabilities/index.ejs]: responsible ~to handle~ of handling ...

**Pull request merged**